src/ducks/Studio/Chart/MetricsExplanation/MetricInsight/MetricInsightCard.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
```

Comment:
@GarageInc This component better to move in components folder. It can be used in other places + it depends on components.